### ISSUE TITLE
Duplicates shown in user interface.

### ISSUE THREAD

**Issue by stefjoosten** [AmpersandTarski/Ampersand#508]
Consider the following interface specification

```
INTERFACE StudentScripts FOR Tutor : '_SESSION'[SESSION]#(I[Account]/\submittor~;submittor) cRud
  BOX<SCOLS>
        [ indiener : I/\submittor~;submittor cRud
        , scripts : submittor~ cRud
          BOX<SCOLS>
              [ opdracht  : assignment cRud
              , bestand   : content cRUd
              , ingediend : submitted cRud
              ]
        ]
```

It produces the following effect: 

![image](https://cloud.githubusercontent.com/assets/275768/17830982/9f07fbc2-66dd-11e6-836e-91b262d87285.png)

In field  "indiener", it is unexpected to see the same atom 3 times.
# Reproducing this bug.

tbd

**Comment by hanjoosten** [comment]
It might be due to the fact that no select DISTINCT is being done, because of optimalisations. In this case, the outer select DISTINCT should be wrapped by the frontend. @Michiel-s , do you agree?

**Comment by Michiel-s** [comment]
This indeed relates to #486, which is not completely resolved and fully tested (yet).

**Comment by hanjoosten** [comment]
@stefjoosten I have fixed the related bug of #486. I have made a pull request of that fix. However, since you still promised a reproduction path (see the original post above), could you test that this fix solves your problem?

**Comment by Michiel-s** [comment]
PR is merged. Issue is solved. @stefjoosten please reopen this issue if your case is not solved.
